server/sonar-server/src/main/java/org/sonar/server/user/UserSession.java

```diff
@@ -63,29 +63,31 @@
 
   /**
    * Ensures that permission is granted to user, otherwise throws a {@link org.sonar.server.exceptions.ForbiddenException}.
-   */
-  UserSession checkPermission(String globalPermission);
-
-  /**
-   * @deprecated Only used by Views plugin.
+  
+   * @deprecated in 6.3 because it doesn't support organizations
+   * @see org.sonar.core.permission.GlobalPermissions
+   * @see #checkIsRoot() for system administrators
+   * @see #checkOrganizationPermission(String, String) for organization members
    */
   @Deprecated
-  UserSession checkGlobalPermission(String globalPermission);
-
-  /**
-   * Ensures that at least one of the global permissions is granted to user. If none are granted,
-   * then throws a {@link org.sonar.server.exceptions.ForbiddenException}.
-   */
-  UserSession checkAnyPermissions(Collection<String> globalPermissions);
+  UserSession checkPermission(String globalPermission);
 
   /**
    * Does the user have the given permission ?
+  
+   * @deprecated in 6.3 because if doesn't support organizations
+   * @see org.sonar.core.permission.GlobalPermissions
+   * @see #isRoot()
+   * @see #hasOrganizationPermission(String, String)
    */
+  @Deprecated
   boolean hasPermission(String globalPermission);
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)